### PR TITLE
Update the build commands for m1 macos

### PR DESCRIPTION
**Pull request by axonasif**
Ref #1884

### CODE REVIEW

File: docs/build-source-code.rst [hunk 1]
```diff
@@ -135,9 +135,18 @@ Build with the following commands:
 
 ::
 
-    cmake -DCMAKE_PREFIX_PATH="$(brew --prefix qt5)" .
+	mkdir -p build && cd build
+    cmake -DCMAKE_PREFIX_PATH=(brew --prefix qt5) -DCMAKE_LISTS_TXT_PATH="$PWD/../CMakeLists.txt" -DCMAKE_LISTS_OR_SETTINGS_JSON=CMakeListsTxtAdvanced -DCMAKE_BUILD_TYPE=Release -DCMAKE_GENERATOR=Ninja -DWITH_TESTS=TRUE ../
```
Comment (Moulick) [inline docs/build-source-code.rst:139]: shouldn't `(brew --prefix qt5)` be `$(brew --prefix qt5)`

Comment (Moulick) [inline docs/build-source-code.rst]: shouldn't `(brew --prefix qt5)` be `$(brew --prefix qt5)`

```suggestion
    cmake -DCMAKE_PREFIX_PATH=$(brew --prefix qt5) -DCMAKE_LISTS_TXT_PATH="$PWD/../CMakeLists.txt" -DCMAKE_LISTS_OR_SETTINGS_JSON=CMakeListsTxtAdvanced -DCMAKE_BUILD_TYPE=Release -DCMAKE_GENERATOR=Ninja -DWITH_TESTS=TRUE ../
```

File: docs/build-source-code.rst [hunk 2]
```diff
@@ -135,9 +135,18 @@ Build with the following commands:
...
     cmake --build .
-    cpack
+    cpack && codesign --force --deep --sign - $PWD/_CPack_Packages/Darwin/DragNDrop/copyq-*-Darwin/CopyQ.app
+	mv _CPack_Packages/Darwin/DragNDrop/copyq-*-Darwin/CopyQ.app .
```
Comment (Moulick) [inline docs/build-source-code.rst:142]: This returns error
```bash
$ mv _CPack_Packages/Darwin/DragNDrop/copyq-*-Darwin/CopyQ.app .
mv: rename _CPack_Packages/Darwin/DragNDrop/copyq-6.1.0-23-g36c1c8a2-Darwin/CopyQ.app to ./CopyQ.app: Directory not empty
```
Seems that the steps above already generate a `CopyQ.app` in this directory, making this move fail

And actually, we can just copy the `$PWD/_CPack_Packages/Darwin/DragNDrop/copyq-*-Darwin/CopyQ.app` right after the codesign to the Applications folder, the rest of the steps seem a bit pointless ?

File: docs/build-source-code.rst (hunk 1 above)
Comment (axonasif) [reply docs/build-source-code.rst]: Oh yeah sorry, I was using fish shell instead of bash, so that's fish syntax. Will update it to bash

**Comment by hluk** [comment]
Thanks for looking into this.

Would it be possible to update CMake files instead or perhaps create a script? The new steps in documentation look very complex.

**Comment by hluk** [comment]
Any updates?

**Comment by Moulick** [comment]
the command `brew tap copyq/kde utils/github/homebrew/` does not work anymore, I do not know why. It gives the error 
```bash
Error: Tap copyq/kde remote mismatch.
/Users/moulick.aggarwal/CopyQ/utils/github/homebrew/ != utils/github/homebrew/
```

**Comment by hluk** [comment]
> the command `brew tap copyq/kde utils/github/homebrew/` does not work anymore, I do not know why. It gives the error
> 
> ```shell
> Error: Tap copyq/kde remote mismatch.
> /Users/moulick.aggarwal/CopyQ/utils/github/homebrew/ != utils/github/homebrew/
> ```

Can you run `brew untap --force copyq/kde` first? (see: https://copyq.readthedocs.io/en/latest/build-source-code.html#building-and-packaging-for-os-x)

**Comment by Moulick** [comment]
> > the command `brew tap copyq/kde utils/github/homebrew/` does not work anymore, I do not know why. It gives the error
> > ```shell
> > Error: Tap copyq/kde remote mismatch.
> > /Users/moulick.aggarwal/CopyQ/utils/github/homebrew/ != utils/github/homebrew/
> > ```
> 
> Can you run `brew untap --force copyq/kde` first? (see: https://copyq.readthedocs.io/en/latest/build-source-code.html#building-and-packaging-for-os-x)

ah yep, that was the issue. Well, not an issue per see, just that I forgot that I had `kf5-*` stuff installed via that folder and did not need to do that again 😓.

**Comment by hluk** [comment]
Closing. M1 support has been merged in #2609.

**Pull request closed without merging**